html_css/intermediate_css/default-styles.md

```diff
@@ -7,19 +7,19 @@ Browsers, by default, inject a little bit of style into your web projects. You m
 
 ### Default Styles
 
-When you did the first HTML Recipe project, you will have noticed that some elements already have a little bit of style applied to them. An `h1`, for instance, is bigger and bolder than regular text. Links (`<a>`) are blue and underlined. Lists (`ul` and `ol`) both have a bunch of padding surrounding them. Browsers accomplish this by inserting a little bit of css into every webpage. ([Here is Chrome's default html stylesheet](https://chromium.googlesource.com/chromium/blink/+/refs/heads/main/Source/core/css/html.css))
+When you did the first HTML Recipes project, you would have noticed that some elements already have a little bit of style applied to them. An `h1`, for instance, is bigger and bolder than regular text. Links (`<a>`) are blue and underlined. Lists (`ul` and `ol`) both have a bunch of padding surrounding them. Browsers accomplish this by inserting a little bit of CSS into every webpage. ([Here is Chrome's default HTML stylesheet](https://chromium.googlesource.com/chromium/blink/+/refs/heads/main/Source/core/css/html.css))
```

Comment:
The changes look good. A couple of suggestions:

1. The text "HTML Recipes project" could link to the Recipes project in our Foundations path.
2. We should remove the angled brackets around the `a` tag to be more consistent .

Reply:
> The changes look good. A couple of suggestions:
> 
>     1. The text "HTML Recipes project" could link to the Recipes project in our Foundations path.
> 
>     2. We should remove the angled brackets around the `a` tag to be more consistent .

I made a new PR with all of the changes there, since I'm not sure how to edit this PR.

Reply:
You would have to make a new commit on the `patch-5` branch of your forked repository. After pushing that commit, it would've showed up here.

Anyways, thanks for making a new PR, closing this now.